### PR TITLE
Meta enrolment in csv course upload

### PR DESCRIPTION
**Pull request by ilyatregubov**
MDL-73852 changes

### CODE REVIEW

File: docs/devupdate.md [hunk 1]
```diff
@@ -11,6 +11,17 @@ import { Since } from '@site/src/components';
 
 This page highlights the important changes that are coming in Moodle 4.4 for developers.
 
+## Meta enrolment support in csv course upload
```
Comment (sarjona) [inline docs/devupdate.md:14]:
```suggestion
## Enrolment

### Meta enrolment support in CSV course upload
```

File: docs/devupdate.md [hunk 2]
```diff
@@ -11,6 +11,17 @@ import { Since } from '@site/src/components';
...
+<Since version="4.4" issueNumber="MDL-73852" />
+
+It is now possible to use meta enrolment when uploading courses using CSV file. Here is an example of csv file that can be used to upload a course with meta enrolment:
+
+```php
+shortname,fullname,category_idnumber,enrolment_1,enrolment_1_metacoursename
+course1,Course 1,CAT1,meta,course2
+```
+
 ## Multiple enrol instances of same type in csv course upload
```
Comment (sarjona) [inline docs/devupdate.md:25]:
```suggestion
### Multiple enrol instances of same type in CSV course upload
```

File: docs/devupdate.md [hunk 3]
```diff
@@ -11,6 +11,17 @@ import { Since } from '@site/src/components';
 
 This page highlights the important changes that are coming in Moodle 4.4 for developers.
 
+## Meta enrolment support in csv course upload
+
+<Since version="4.4" issueNumber="MDL-73852" />
+
+It is now possible to use meta enrolment when uploading courses using CSV file. Here is an example of csv file that can be used to upload a course with meta enrolment:
```
Comment (sarjona) [inline docs/devupdate.md:18]:
```suggestion
It is now possible to use meta enrolment when uploading courses using CSV file. Here is an example of CSV file that can be used to upload a course with meta enrolment:
```

File: docs/devupdate.md [hunk 4]
```diff
@@ -11,6 +11,17 @@ import { Since } from '@site/src/components';
...
+
+```php
```
Comment (sarjona) [inline docs/devupdate.md:20]:
```suggestion
```php title="Example of CSV for meta enrolment"
```

**Review by sarjona — CHANGES_REQUESTED** [review]
Hi @ilyatregubov!
Thanks a lot for preparing a patch for this. As you'll see, I added a few minor proposals to improve it :-)

**Comment by ilyatregubov** [comment]
All issues have been resolved. Just need someone to approve

**Review by sarjona — APPROVED** [review]
Hi @ilyatregubov!
Thanks for reviewing the patch. I'm happy to approve and merge it. Well done!

**Pull request merged**